omigami/cli_options.py

```diff
@@ -1,16 +1,15 @@
 import click
 
-from omigami.config import GNPS_URIS, STORAGE_ROOT
+from omigami.config import STORAGE_ROOT
 
 common_flow_options = [
     click.option(
-        "--image", "-i", type=str, required=True, help="Image used to build the flow"
-    ),
-    click.option(
-        "--project-name",
-        "-p",
-        help="Name of the project. This is used as identification by Prefect",
-        required=True,
+        "--image",
+        "-i",
+        type=str,
+        required=False,
+        help="Image used to build the flow",
```

Comment:
can we add, here or on the readme, that this is required for running with prefect server?